Enhancement: Add support for configurable ports in Docker Compose #46

GJS2162 opened the pull request:
This pull request updates the Docker Compose configuration to support dynamic port configuration using environment variables. The changes include modifying the Docker Compose file to utilize environment variables for specifying ports, enhancing flexibility and customization options. Additionally, a Python script is provided to automate the modification process. Issue #46 tracks the implementation of this enhancement.

GJS2162 commented:
> Can you fix the pre-commit tests ?

Sure, I’ll look into it. In the meantime, could you provide me with some additional information or resources that might help?

Mr-Sunglasses commented:
> > Can you fix the pre-commit tests ?
> 
> Sure, I’ll look into it. In the meantime, could you provide me with some additional information or resources that might help?

Ref: https://pre-commit.com/#usage

GJS2162 commented:
> Can you fix the pre-commit tests ?

I removed the script which I have added, I think now it will work.
pre-commit job were running for this commit : [11ac865](https://github.com/tushar5526/sarthi/pull/52/commits/11ac865f7a93504b145271460a07ac1f0e820f9c)

GJS2162 commented:
> > > Can you fix the pre-commit tests ?
> > 
> > 
> > Sure, I’ll look into it. In the meantime, could you provide me with some additional information or resources that might help?
> 
> Ref: https://pre-commit.com/#usage

Thank you Kanishk for the resource.
